src/locale/mr.js

```diff
@@ -52,19 +89,19 @@ export default moment.defineLocale('mr', {
         sameElse : 'L'
     },
     relativeTime : {
-        future : '%s नंतर',
-        past : '%s पूर्वी',
-        s : 'सेकंद',
-        m: 'एक मिनिट',
-        mm: '%d मिनिटे',
-        h : 'एक तास',
-        hh : '%d तास',
-        d : 'एक दिवस',
-        dd : '%d दिवस',
-        M : 'एक महिना',
-        MM : '%d महिने',
-        y : 'एक वर्ष',
-        yy : '%d वर्षे'
+        future: '%sमध्ये',
```

Comment:
I don't think त == च्या अात

त is from Marathi विभक्ती प्रत्यय सप्तमी - त इ अा
च्या अात makes it षष्ठी + अात

These 2 त have no relation.